src/runtime_src/core/pcie/windows/mgmt.cpp

```diff
@@ -259,14 +259,14 @@ struct mgmt
     DWORD bytes = 0;
 
     auto status = DeviceIoControl
-    (m_hdl,
-    XCLMGMT_OID_PRP_ICAP_PROGRAM_AXLF,
-    (PUCHAR)buffer,
-    buffSize,
-    nullptr,
-    0,
-    &bytes,
-    nullptr);
+        (m_hdl,
+        XCLMGMT_OID_PRP_ICAP_PROGRAM_AXLF, //ioctl code
```

Comment:
Yes, this does increase the verbosity and limit some flexibility, but it will assist future developers in debugging (if needed) and understanding the code base.